Installation/cmake/modules/FindSuiteSparse.cmake

```diff
@@ -0,0 +1,437 @@
+## CMake file to locate SuiteSparse and its useful composite projects
```

Comment:
It's not shipped with the library, it's from another project ( https://github.com/jlblancoc/suitesparse-metis-for-windows ). Provided I copy paste their licence.md instead of the pointer to the licence.md that they had, it's alright to keep?